Prolog/models/localStorage.pl

```diff
@@ -1,3 +1,12 @@
-:- module(localStorage,[testMethodLocalStorage/1]).
+:- module(localStorage,[readHighestScore/1, saveHighestScore/1]).
 
-testMethodLocalStorage(Param):- write(Param).
\ No newline at end of file
+saveHighestScore(Score):- open("score.txt",write,Stream),
+         				  write(Stream,Score),  
+         				  write(Stream,"."), 
+         				  nl(Stream),
+         				  close(Stream).
+
+readHighestScore(Score):- (exists_file("Score.txt") ->  open("score.txt",read,HighestScore),

```

Comment:
Did you mean `exists_file("score.txt")` (with lowercase "s")?

Reply:
Yes, my bad.

Reply:
Already submitted an update changing to lowercase